docs/background-clip.md

```diff
@@ -14,18 +14,18 @@ Utilities for controlling the bounding box of an element's background.
 Use the `bg-clip-{keyword}` utilities to control the bounding box of an element’s background.
 
 <container>
-  <div class="flex flex-col sm:flex-row items-center justify-center gap-32">
-    <div class="flex flex-col items-center shrink-0">
-      <p class="pd-text-slate-500 pd-font-mono text-center mb-16 dark:pd-text-slate-400">bg-clip-border</p>
-      <div class="w-80 h-80 bg-clip-border p-16 rounded-8 pd-shadow-lg pd-bg-indigo-500 border-4 pd-border-white/50 border-dashed pd-font-mono pd-font-extrabold pd-text-sm pd-text-white flex justify-center items-center"></div>
+  <div class="flex sm:grid sm:grid-cols-3 sm:gap-24 sm:justify-around">
+    <div class="flex flex-col justify-self-center place-items-center">
+      <div class="mb-16 pd-font-mono">bg-clip-border</div>
+      <div class="w-128 h-128 bg-clip-border p-16 rounded-8 pd-shadow-lg pd-bg-indigo-500 border-8 pd-border-white/50 border-dashed pd-font-mono pd-font-extrabold pd-text-sm pd-text-white flex justify-center items-center"></div>
     </div>
-    <div class="flex flex-col items-center shrink-0">
-      <p class="pd-text-slate-500 pd-font-mono text-center mb-16 dark:pd-text-slate-400">bg-clip-padding</p>
-      <div class="w-80 h-80 bg-clip-padding p-16 rounded-8 pd-shadow-lg pd-bg-indigo-500 border-4 pd-border-indigo-500/50 border-dashed pd-font-mono pd-font-extrabold pd-text-sm pd-text-white flex justify-center items-center"></div>
+    <div class="flex flex-col justify-self-center place-items-center">
+      <div class="mb-16 pd-font-mono">bg-clip-padding</div>
+      <div class="w-128 h-128 bg-clip-padding p-16 rounded-8 pd-shadow-lg pd-bg-indigo-500 border-8 pd-border-indigo-500/50 border-dashed pd-font-mono pd-font-extrabold pd-text-sm pd-text-white flex justify-center items-center"></div>
     </div>
-    <div class="flex flex-col items-center shrink-0">
-      <p class="pd-text-slate-500 pd-font-mono text-center mb-16 dark:pd-text-slate-400">bg-clip-content</p>
-      <div class="w-80 h-80 bg-clip-content p-16 rounded-8 pd-bg-indigo-500 border-4 pd-border-indigo-500/50 border-dashed pd-font-mono pd-font-extrabold pd-text-sm pd-text-white flex justify-center items-center"></div>
+    <div class="flex flex-col justify-self-center place-items-center">
+      <div class="mb-16 pd-font-mono">bg-clip-content</div>
```

Comment:
Does it make any difference for a11y if this is a div or a p? 🤔  It should perhaps be a heading... since it is a label for the example below... but I'm not sure what this should be in regards to a11y? 🤷

Reply:
A table sounds like the most reasonable solution here... together with descriptions, as you say, as this is examples of purely visual things. But this should be done in a separate tasks on all pages, so that it is consistent across the docs, or what do you think? (That will be a huge task.)

This is for sure NOT a paragraph though. From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p :

> The <p> HTML element represents a paragraph. Paragraphs are usually represented in visual media as blocks of text separated from adjacent blocks by blank lines and/or first-line indentation, but HTML paragraphs can be any structural grouping of related content, such as images or form fields.

And in my opinion we should never see Tailwind as a good example that we should follow.

Reply:
I have changed these to headings with simplified sr-only descriptions for now. We will then need a separate ticket for a11y in docs to go through it all.

Reply:
If there's no plan for rewriting the whole docs to something else in the near future, then I guess this Jira ticket should be looked into: https://nmp-jira.atlassian.net/browse/WARP-263